.circleci/config.yml

```diff
@@ -298,7 +298,11 @@ workflows:
       # expect the ceremony to work continuously), and remove it once
       # the ceremony is for historical archive only (e.g. the
       # ceremony is done).
-      - just_simulate_sc_rehearsal_1
+      #
+      # We skip rehearsal 1 because we already have completed this rehearsal (https://github.com/ethereum-optimism/superchain-ops/pull/459),
+      # and now it fails with stack too deep after https://github.com/ethereum-optimism/superchain-ops/pull/528.
+      # We wll need to rewrite the rehearsals with the new superchain-ops structure anyway, so this is ok.
+      # - just_simulate_sc_rehearsal_1
```

Comment:
Tracking in https://github.com/ethereum-optimism/superchain-ops/issues/529 and added that as a non-blocker to https://github.com/ethereum-optimism/superchain-ops/issues/396